Gemfile

```diff
@@ -10,11 +10,11 @@ gem 'delayed_job_active_record'
 
 # Metadata presenter - if you need to be on development you can uncomment
 # one of these lines:
-# gem 'metadata_presenter',
-#     github: 'ministryofjustice/fb-metadata-presenter',
-#     branch: 'page-with-component'
+gem 'metadata_presenter',
+    github: 'ministryofjustice/fb-metadata-presenter',
+    branch: 'add-branching-title'
 # gem 'metadata_presenter', path: '../fb-metadata-presenter'
-gem 'metadata_presenter', '2.2.0'
+# gem 'metadata_presenter', '2.2.0'
```

Comment:
can swap this back to `2.2.0`

Reply:
I was waiting for the 2.3.0 to be released. All good now 🚀